app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,33 @@
+class SessionsController < ApplicationController
+  before_action :load_user, only: :create
+  before_action :check_authentication, only: :create
+
+  # Handles user login and session creation
+  def create
```

Comment:
```suggestion
  # POST: /login
  def create
```
Comment router khi truy cập, k phải mô tả. Thêm tương tự cho các action khác